src/app/services/local-storage.service.ts

```diff
@@ -11,6 +11,10 @@ export class LocalStorageService {
         return localStorage.getItem(AppValues.LOCAL_STORAGE_LOGIN_CALLBACK_URL);
     }
 
+    public get accountId(): string | null {
+        return localStorage.getItem(AppValues.LOCAL_STORAGE_ACCOUNT_ID);
+    }
```

Comment:
Make a property or method that does not deal with `null`. Throw an exception, if account is undefined.
It would look much more natural during operations, as you won't need dummy `""` default value, 
also we will have a stricter check for the actions that do not allow anonymous users.

Reply:
Fixed, added new exception